### PR TITLE
18536 Additions for amalgamations + misc cleanup

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#18536

**I highly recommend that you review commits 1-3 independently, since the last commit (Storybook update) has 169 changed files that you don't really have to review. 😛**

*Description of changes:*

Commit 1:
- added Amalgamation Types enum
- added amalgamation filing code
- added amalgamation filing name
- added amalgamation filing type

Commit 2:
- misc import style cleanup
- now import CorpTypeCd from module (not enums/)
- added missing import to interfaces package.json

Commit 3:
- updated Lerna dependencies

Commit 4:
- updated Storybook

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/enums/filing-codes.ts [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 export enum FilingCodes {
   ADDRESS_CHANGE_BC = 'BCADD', // BCOMP - Change of Address
   ADDRESS_CHANGE_OT = 'OTADD', // Others - Change of Address
   AGM_EXTENSION = 'AGMDT',
   AGM_LOCATION_CHANGE = 'AGMLC',
   AFFIDAVIT = 'AFDVT',
   ALTERATION = 'ALTER',
-  ALTERATION_BC_TO_ULC = 'NOALU', // More expensive for BC to ULC.
+  ALTERATION_BC_TO_ULC = 'NOALU', // different fee for BC to ULC
+  AMALGAMATION = 'AMALG',
```
Comment (severinbeauvais) [inline src/enums/filing-codes.ts:10]: I'm still waiting for confirmation on this (or multiple) amalgamation fee code(s).

Comment (JazzarKarim) [reply src/enums/filing-codes.ts:10]: We might have a different fee code depending on the amalgamation type (short, regular etc...)?

**Review by JazzarKarim — APPROVED** [review]
LGTM! 👍

**Review by ketaki-deodhar — APPROVED** [review]
👍

File: src/enums/filing-codes.ts (hunk 1 above)
Comment (severinbeauvais) [reply src/enums/filing-codes.ts]: We now have 3 different filing codes (reg, horiz, vert). Resolving this conversation.

**Pull request merged**